### ISSUE TITLE
关于CrossEntropyLoss_LSR函数的一些问题

### ISSUE THREAD

**Issue by AutWind** [songyouwei/ABSA-PyTorch#81]
你好。在aen.py这个模型文件中，CrossEntropyLoss_LSR函数实现了损失的计算。但我发现它和论文中的描述有些不一致。在论文中，你是基于平滑调整后的真实标签与模型预测结果计算交叉熵，然后加上基于KL散度计算得到LSR项以及L2正则化项，作为损失函数。但是在代码中，却只有真实标签的平滑调整与交叉熵计算，即：
        one_hot_label = self._toOneHot_smooth(label, b, c).to(self.device)
        loss = torch.sum(-one_hot_label * self.logSoftmax(pre), dim=1)
我发现这里的loss并没有加上LSR项与L2正则化项，请问为什么会不一致呢？

**Comment by songyouwei** [comment]
是一致的。
- _toOneHot_smooth得到的label，数学上等价于LSR项
- L2正则项体现在optimizer的weight_decay参数上

**Comment by AutWind** [comment]
> 是一致的。
> 
> * _toOneHot_smooth得到的label，数学上等价于LSR项
> * L2正则项体现在optimizer的weight_decay参数上

好的，我明白了。感谢你的解答！
